open-commerce-search-api/src/main/java/de/cxp/ocs/model/result/SearchResult.java

```diff
@@ -36,15 +26,6 @@
 	 */
 	public SearchParams params;
```

Comment:
Would it make sense to move (or copy?) the SearchParams to the ResultSlice? 
Because it would describe the exact parameters to retrieve that particular result slice.